src/site/site.xml

```diff
@@ -338,6 +318,27 @@
         </item>
       </item>
 
+      <item name="Filters" href="filters/index.html" collapse="true">
```

Comment:
Why are these collapsed by default when other single level items are expanded?

Reply:
All categories are collapsed - `Annotations`, `Blocks`, etc...

The idea was to keep the original length of the menu. I am not sure if making it longer would be good. Especially on mobile.

Reply:
We moved these out of checks since they are not the same, so this is not a good comparison :) If you look at other siblings of these two that are only single depth, they are already expanded.

Reply:
I am fine with model: expanding of one collapse others.

Reply:
we will vote at https://github.com/checkstyle/checkstyle/issues/13306